codingstandards/continuousintegration.md

```diff
@@ -0,0 +1,55 @@
+
+## Install vs. Update
+For each database using the CI design there are two levels of versioning that need to be changed. They are `install` and `update`.
+
+### Install
+The `install` changes are all the changes needed to create the database from scratch. This will include all table declorations and stored procedures. The `install` changes will be applied whenever a devoloper wants to create a database. This reqires the `install` directory to have **the current state of the database**. All changes to the database will modify the `install` directory.
+
+### Update
+The `update` changes are all the changes needed to get the database from point **A** to point **B**. The `update` changes are versioned with a unique modifier so that the database knows what updates it has completed and which ones it needs to run. The PROD database will run the `update` changes whenever a new change is released. 
+
+## Performing a Change
+When making a change to a database using the CI design follow these steps.
+
+### 1. Modify Install directory
+
+Modifiy the `*ProjectName*/Database/Install` directory to reflect the current state of the database. The `Install` directory has multiple subdirectories for the different components of the database. 
+
+![Install folder](/images/LegacyWebInstallFolder.png)
+
+* The `Functions` directory stores functions.
+* The `Routines` directory stores Stored Procedures.
+* The `Security` directory stores Schemas.
+* The `Tables` directory stores Table Declorations.
+* The `Types` directory stores Type Descriptions.
+
+Inside the `Install` directory is a C# file that controls what scripts are loaded on a database install. 
+
+![Migration file](/images/LegacyWebMigrationFile.png)
+
+A script is loaded with the command `Execute.EmbeddedScript("Database.Install.Tables.dbo.Customer.sql");`. This command takes as a parameter the path of the script. So the `Customer.sql` script is located at `*ProjectName*/Database/Install/Tables/dbo.Customer.sql`. 
+
+**When creating a new file or changing a filename make sure to add the script to the C# migration file.** When changing a file verify that it is in the C# migration file. 
+
+> Note: All scripts should be rerunnable. i.e. if a create table script is ran on a database that already has that table, that command will be ignored.
+
+### 2. Modify Update directory
+
+The `update` directory keeps track of changes that will advance the database from the previous state to the current state. Each update will be stored in it's own subdirectory. 
+
+![Update folder](/images/LegacyWebUpdateFolder.png)
+
+**Table changes will include SQL scripts** in the update subdirectories while **all other changes will only have the C# migration file**. 
+
```

Comment:
Also point out that the Build Action of new SQL scripts should be "Embedded Resource".

Reply:
Done.